Source/FLHookPluginSDK/headers/plugin.h

```diff
@@ -365,11 +368,37 @@ struct CUSTOM_JUMP_STRUCT
 	uint iSystemID;
 };
 
+struct CUSTOM_JUMP_CALLOUT_STRUCT
+{
+	uint iClientID;
+	uint iSystemID;
+	Vector pos;
+	Matrix ori;
+	uint jumpType = 0;
```

Comment:
Unused?

Reply:
It is used in the PlayerController versions of this callout. Again, wanted to avoid code duplication across branches.